src/mruby-zest/example/TextLine.qml

```diff
@@ -3,6 +3,7 @@ Widget {
     property Function whenValue: nil
     property Object   valueRef:  nil
     property Bool     upcase:    true
+    // TODO: this property is not used by any calling code. Delete?
```

Comment:
I think this might have been copied incorrectly from use in a file selector context (aka it expect the output to have some extension .txt, .xmz, etc). I'm guessing your comment is right.